code/pizzabot.js

```diff
@@ -1,7 +1,47 @@
+
 const vegetarian = "Vegetarian Pizza"
 const hawaiian = "Hawaiian Pizza"
 const pepperoni = "Pepperoni Pizza"
+const margherita = "Margherita Pizza"
 
 const pizzaPrice = 80
 
-//Put your Javscript code here:
+
+const calculateTotalCost = (orderQuantity) => {
+    orderTotal = (orderQuantity * pizzaPrice)
+}
+
+const calculateCookingTime = (orderQuantity) => {
+    if (orderQuantity > 0 && orderQuantity <= 2) {
+        orderTime = 10
+    } else if (orderQuantity >= 3 && orderQuantity <= 5) {
+        orderTime = 15
+    } else if (orderQuantity >= 6) {
+        orderTime = 20
+    }
+}
+
+let validateOrderName = (orderName, orderQuantity) => {
+    event.preventDefault();
```

Comment:
So are we using semicolons in this project or not?
It's up to you whether you use them or not, what's most important is consistence - you either use it everywhere, or not at all